Patch extracted from this tails-iuk commit:
commit bc6aad74b1ba49c942a504a495f87bf0cfa2ce33
Author: Tails developers <tails@boum.org>
Date:   Sun Dec 7 13:37:03 2014 +0000

    Add an indirection layer rather than hard-coding the CA to verify the website certificate against.
    
    The tails-iuk.pem CA bundle is created at build time, in the
    bugfix/handle-website-CA-change branch of our main Git repository.

--- chroot.orig/usr/share/perl5/Tails/IUK/UpgradeDescriptionFile/Download.pm
+++ chroot/usr/share/perl5/Tails/IUK/UpgradeDescriptionFile/Download.pm
@@ -105,7 +105,7 @@ method _build_curl_opts {
     }
     else {
         my $cafile = $ENV{HTTPS_CA_FILE};
-        $cafile  //= '/etc/ssl/certs/UTN_USERFirst_Hardware_Root_CA.pem';
+        $cafile  //= '/usr/local/etc/ssl/certs/tails-iuk.pem';
         push @opts, CURLOPT_SSL_VERIFYHOST,  2;
         push @opts, CURLOPT_SSL_VERIFYPEER,  1;
         push @opts, CURLOPT_CAINFO,          $cafile;
